src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -111,6 +113,8 @@ public class OrganizationService : IOrganizationService
         _providerUserRepository = providerUserRepository;
         _countNewSmSeatsRequiredQuery = countNewSmSeatsRequiredQuery;
         _updateSecretsManagerSubscriptionCommand = updateSecretsManagerSubscriptionCommand;
+        _orgUserInviteTokenableFactory = orgUserInviteTokenableFactory;
+        _orgUserInviteTokenDataFactory = orgUserInviteTokenDataFactory;
```

Comment:
typo: Looks like the line got duplicated

Reply:
`_orgUserInviteTokenableFactory` !== `_orgUserInviteTokenDataFactory `

I thought it was a duplicate as well, and after removing it, I was confused why I started having test failures.

Reply:
🤦 whoops, sorry about that!

Reply:
haha all good!